osc/commandline.py

```diff
@@ -7571,24 +7571,45 @@ def do_service(self, subcmd, opts, *args):
         """
         # disabledrun and localrun exists as well, but are considered to be obsolete
 
-        args = slash_split(args)
-        project = package = singleservice = mode = None
         apiurl = self.get_api_url()
+
         remote_commands = ('remoterun', 'rr', 'merge', 'wait')
 
+        try:
+            store = osc_store.Store(Path.cwd())
+            store.assert_package()
```

Comment:
Fix is in https://src.opensuse.org/mcepl_tools/osc/commit/8dbe5a5c8eb7a255e89a672dd08c5cb44f065da7 (`assert_package` -> `assert_is_package`).

If we merge this as it is, then the situation won’t get worse, just not that much better, and it seems we still need both package `.osc/` and the project ones as well. So, unless better option is created (and that will be bigger undertaking), I would accept it as at-least-it-works-somehow.